src/components/value-list/value-list.tsx

```diff
@@ -335,6 +393,43 @@ export class ValueList<
     return type;
   }
 
+  updateScreenReaderText(assertiveText: string): void {
+    //code to update text to our aria-live span element
+    const screenReaderElement = this.el.shadowRoot.querySelector(".assistive-text");
+    screenReaderElement.textContent = assertiveText;
+  }
+
+  updateHandleAriaLabel(handleElement: HTMLSpanElement, assertiveText: string): void {
+    handleElement.ariaLabel = assertiveText;
+  }
+
+  getScreenReaderText(item: HTMLCalciteValueListItemElement, status: DragStatus): string {
+    const { items } = this;
+    const total = items.length;
+    const position = getItemIndex(this, item) + 1;
+
+    switch (status) {
+      case "start":
+        return this.intlDragHandleStart
+          ? this.intlDragHandleStart
+          : `press space and use arrow keys to re-order content. current position ${position} of ${total}`;
```

Comment:
Should all these strings be stored in a separate file? Like a resources file?

Reply:
The strings contain variables `${position}` , `${total}` which cant be stored in  a resource file.

Reply:
nvm. gotcha